conf/messages

```diff
@@ -640,6 +640,8 @@ fh.uniqueTaxpayerReference_value.error.pattern=Enter a valid Self Assessment Uni
 fh.uniqueTaxpayerReference_value.error.required=Enter a Self Assessment Unique Taxpayer Reference (UTR)
 fh.businessPartners.partnership.tradingName.title = Does the partnership use a trading name that is different from its registered name?
 fh.businessPartners.partnership.tradingName.heading = Does {0} use a trading name that is different from its registered name ?
+fh.businessPartners.corporateBody.tradingName.title = Does the corporate body use a trading name that is different from its registered name?
```

Comment:
It appears that the - Business partners - is missing from the title. 
<img width="586" alt="Screenshot 2023-04-20 at 11 32 41" src="https://user-images.githubusercontent.com/45938748/233340947-39f6b086-5850-403d-8732-ba6c58e3ac50.png">